cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/datastore/callables/AttachmentsForRevisionCallable.java

```diff
@@ -0,0 +1,70 @@
+package com.cloudant.sync.datastore.callables;
```

Comment:
Thanks, although it pains me to say that the guidelines are to use (C) or © I'm not sure what isn't correct about the lowercase (c) - admittedly we have not fixed that throughout the library I was hoping to just get them right as we go through.